src/main/_docs/admin-guide/admin-cli.md

```diff
@@ -7,8 +7,59 @@ permalink: /:categories/cli/
 ---
 {% include base.html %}
 
+When you run the image `codenvy/cli` that is a special Docker image which is the CLI launcher. It contains the CLI which has the functions to manage a Codenvy installation. The CLI, in turn, launches Codenvy from a configuration, which launches a number of other containers which actually run the Codenvy servers. 
```

Comment:
The paragraph didn't make to much sense to me. I rewrote it to the following. Feel free to edit this if you would like to use it.

Running Docker image `codenvy/cli` executes Codenvy's CLI launcher from within a docker container. The CLI launcher provides various functions and utilities for Codenvy such as a startup utility. The CLI launcher startup utility uses a configuration file. The startup utility launches a number of Docker containers that work collectively to provide Codenvy's  server. Codenvy server configuration and data files/folders are stored on the local host via volume mounts which allows the local host to start another Codenvy server with the same configuration and data after the existing Codenvy server is removed/shutdown.